### PR TITLE
auth test cases

### PR DESCRIPTION
**Pull request by Subhransupanda2000**
Screenshot (566)

### CODE REVIEW

File: src/main/java/com/example/bookstoreapp/utils/AppUtils.java [hunk 1]
```diff
@@ -11,11 +11,23 @@ public class AppUtils {
   public static String getEncodedString(String rawStr){
     return new String(Base64.getEncoder().encode(rawStr.getBytes(StandardCharsets.UTF_8)));
   }
```
Comment (nginx500) [inline src/main/java/com/example/bookstoreapp/utils/AppUtils.java:13]: formatting of code required.

File: src/main/java/com/example/bookstoreapp/utils/AppUtils.java [hunk 2]
```diff
@@ -11,11 +11,23 @@ public class AppUtils {
...
+  public static String getDecodedString(String base64Str) {
```
Comment (nginx500) [inline src/main/java/com/example/bookstoreapp/utils/AppUtils.java:14]: implementation should not change as it is a util class.

**Review by nginx500 — CHANGES_REQUESTED** [review]
do changes

File: src/test/java/com/example/bookstoreapp/auth/AuthServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+        // Then
+        assertFalse(result);
+        verify(authCacheClient, never()).remove(anyString(), anyString());
+    }
+
+    @Test
+    void testSetUserContext() {
+        // Given
+        String tokenStr = UUID.randomUUID().toString();
```
Comment (nginx500) [inline src/test/java/com/example/bookstoreapp/auth/AuthServiceImplTest.java:85]: convert to base64 string

File: src/test/java/com/example/bookstoreapp/auth/AuthServiceImplTest.java [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.test.context.TestPropertySource;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (nginx500) [inline src/test/java/com/example/bookstoreapp/auth/AuthServiceImplTest.java:17]: remove wildcard import statements.

**Review by rkpatra201 — CHANGES_REQUESTED** [review]

**Comment by rkpatra201** [comment]
image

code coverage is 100%

**Review by rkpatra201 — APPROVED** [review]

**Comment by rkpatra201** [comment]
> do changes

done

**Pull request merged**